tests/test_supabase.py

```
@@ -1,6 +1,6 @@
-from ..backend.app.model.supabase.models import User, Interaction, CodeChunk, Repository
+from ..backend.app.models.database.supabase import User, Interaction, CodeChunk, Repository
```

Pylint (3.3.7)

[error] Attempted relative import beyond top-level package

(E0402)